products/gstreamer.md

```diff
@@ -0,0 +1,81 @@
+---
+title: GStreamer
+category: framework
+iconSlug: gstreamer
+permalink: /gstreamer
+versionCommand: gst-inspect-1.0 --version
+releasePolicyLink: https://gstreamer.freedesktop.org/
+changelogTemplate: https://gstreamer.freedesktop.org/data/pkg/src/1.11.90/{{"__RELEASE_CYCLE__"}}
+releaseLabel: "__RELEASE_CYCLE__"
+activeSupportColumn: false
+releaseDateColumn: true
+eolColumn: Supported
+
+auto:
+-   git: https://gitlab.freedesktop.org/gstreamer/gstreamer.git
+    regex: '^(?<major>[1-9]\d*)\.(?<minor>0|[1-9]\d*)\.?(?<patch>0|[1-9]\d*)?$'
+
+releases:
+-   releaseCycle: "1.22"
+    releaseDate: 2023-01-23
+    eol: false
+    latest: "1.22.2"
+    latestReleaseDate: 2023-04-11
+
+-   releaseCycle: "1.20"
+    releaseDate: 2022-02-03
+    latest: "1.20.6"
+
+-   releaseCycle: "1.18"
+    releaseDate: 2020-09-08
+    latest: "1.18.6"
+
+-   releaseCycle: "1.16"
+    releaseDate: 2019-04-19
+    latest: "1.16.3"
+
+-   releaseCycle: "1.14"
+    releaseDate: 2018-03-19
+    latest: "1.14.5"
+
+-   releaseCycle: "1.12"
+    releaseDate: 2017-05-04
+    latest: "1.12.5"
+
+-   releaseCycle: "1.10"
+    releaseDate: 2016-11-01
+    latest: "1.10.5"
+
+-   releaseCycle: "1.8"
+    releaseDate: 2016-03-24
+    latest: "1.8.3"
+
+-   releaseCycle: "1.6"
+    releaseDate: 2015-09-25
+    latest: "1.6.4"
+
+-   releaseCycle: "1.4"
+    releaseDate: 2014-07-19
+    latest: "1.4.5"
+
+-   releaseCycle: "1.2"
+    releaseDate: 2013-09-24
+    latest: "1.2.4"
+
+-   releaseCycle: "1.0"
+    releaseDate: 2012-09-24
+    latest: "1.0.10"
+
+---
+
+> [GStreamer](https://gstreamer.freedesktop.org/) is a pipeline-based
+> multimedia framework that links together a wide variety of media
+> processing systems to complete complex workflows. For instance,
+> GStreamer can be used to build a system that reads files in one
+> format, processes them, and exports them in another. The formats and
+> processes can be changed in a plug and play fashion.
+
+GStreamer follows [Semantic Versioning](https://semver.org/). The
+support and EOL policy is not clearly defined but, looking at the
+releases history, there is at least a `x.y` release every year which
+is supported for at least a year with important bug fixes.
```

Comment:
Maybe it can be noted that only the even minor release numbers are considered ready for production (if we have a link to the GStreamer website that document that it would be great).

Reply:
I will use text from https://gstreamer.freedesktop.org/releases/gstreamer/1.4.0.html

> The versioning scheme that is used in general is that 1.x.y is API and ABI backwards compatible with previous 1.x.y releases. If x is an even number it is a stable release series and all releases in this series will only contain important bugfixes, e.g. the 1.0 series with 1.0.7. If x is odd it is a development release series that will lead to the next stable release series 1.x+1 and contains new features and bigger changes. During the development release series, new API can still change.